lbry/tests/unit/database/test_SQLiteStorage.py

```diff
@@ -247,3 +253,13 @@ class ContentClaimStorageTests(StorageTest):
         current_claim_info = await self.storage.get_content_claim(stream_hash)
         # this should still be the previous update
         self.assertDictEqual(current_claim_info, update_info)
+
+
+class UpdatePeersTest(StorageTest):
+    async def test_update_get_peers(self):
+        node_id = hashlib.sha384(os.urandom(32)).digest()
```

Comment:
generally it's best to avoid using random numbers in tests since they're not repeatable and can be slow if generating lots of them